tests/tracer/test_logger.py

```diff
@@ -335,3 +335,29 @@ def test_logger_does_not_add_handler_when_configured():
     ddtrace_logger = logging.getLogger("ddtrace")
     assert len(ddtrace_logger.handlers) == 0
     assert ddtrace_logger.handlers == []
+
+
+def test_logger_log_level_from_env(monkeypatch):
+    monkeypatch.setenv("_DD_TESTING_DEBUG_LOG_LEVEL", "DEBUG")
+    monkeypatch.setenv("_DD_TESTING_WARNING_LOG_LEVEL", "WARNING")
+    monkeypatch.setenv("_DD_PACKAGE_WITH_UNDERSCORE_SUBMODULE_LOG_LEVEL", "ERROR")
+
+    import ddtrace.internal.logger as dd_logger
+
+    original_trie = dd_logger.LOG_LEVEL_TRIE
+    dd_logger.LOG_LEVEL_TRIE = dd_logger.LoggerPrefix.build_trie()
```

Comment:
```suggestion
    with mock.patch.object(dd_logger, "LOG_LEVEL_TRIE", dd_logger.LoggerPrefix.build_trie())):
```

?

basically same-same, but 🤷🏻 ?